shared/typeinference/codeql/typeinference/internal/TypeInference.qll

```diff
@@ -717,7 +717,7 @@ module Make1<LocationSig Location, InputSig1<Location> Input1> {
     }
 
     /** Provides logic related to base types. */
-    private module BaseTypes {
+    module BaseTypes {
```

Comment:
Making `BaseTypes` public was the simples way to get the needed predicates. We could also more surgically export the needed functions, but I'm not sure if that's worth the trouble.